meteomatics/rounding.py

```diff
@@ -1,47 +1,59 @@
 # -*- coding: utf-8 -*-
 
 decimal_places = {
-    "cm":       1,  # centimeters (snow height)
+    "bft":      0,  # Beauforts (wind)
+    "cm":      -1,  # centimeters (snow height, hail)
     "C":        1,  # temperature
     "d":       -1,  # days AND degrees...
     "dn":       5,  # datenums
-    "eur":      1,  # euro
```

Comment:
sure, that this is no longer needed?

Reply:
I compared it with the list in https://gitlab.meteomatics.com/LIB/mm_cpp_lib/-/blob/master/include/Meteomatics_MeteoCacheGlobals.h#L227 and it isn't included there. 

I checked the cache parameter list and couldn't find any parameter that uses the unit eur.

Reply:
I would still keep it. I know we had it in use for the MarketAtlas and there might be other Energy-Production related products that might use it.